Dockerfile-alpine.template

```diff
@@ -1,3 +1,4 @@
+{{ def llvmver: "15" -}}
```

Comment:
We should move this down right above `DOCKER_PG_LLVM_DEPS` :eyes:

Reply:
Done